src/components/Editor/index.js

```diff
@@ -500,10 +502,13 @@ class Editor extends PureComponent<Props, State> {
     }
 
     if (selectedSource) {
+      const hasJsx = getSymbols(selectedSource);
```

Comment:
Does this mean that, if symbols exist for the selected source, then it has JSX? Not sure i understand this...

Reply:
My bad you're right I got confused with another thing Thanks for the review!